File: bench/tasks/12-revenue/go/main.go
package main

import "fmt"

func main() {
	revenue := []int{240, 175, 289, 150, 225, 199, 180, 178}
	total, highSum, highCount, budgetCount := 0, 0, 0, 0
	for _, r := range revenue {
		total += r
		if r >= 200 {
			highCount++
			highSum += r
		} else {
			budgetCount++
		}
	}
	fmt.Println("Total revenue: " + fmt.Sprint(total))
	fmt.Println("Premium orders (200+): " + fmt.Sprint(highCount))
	fmt.Println("Premium revenue: " + fmt.Sprint(highSum))
	fmt.Println("Budget orders: " + fmt.Sprint(budgetCount))
}

File: bench/tasks/12-revenue/python/main.py
revenue = [240, 175, 289, 150, 225, 199, 180, 178]
total = high_sum = high_count = budget_count = 0
for r in revenue:
    total += r
    if r >= 200:
        high_count += 1
        high_sum += r
    else:
        budget_count += 1
print("Total revenue: " + str(total))
print("Premium orders (200+): " + str(high_count))
print("Premium revenue: " + str(high_sum))
print("Budget orders: " + str(budget_count))
